components/molecule/table/src/index.js

```diff
@@ -0,0 +1,127 @@
+import React, {lazy, Suspense} from 'react'
+import PropTypes from 'prop-types'
+
+const Table = lazy(() => import('antd/lib/table'))
+
+const BASE_CLASS = `sui-MoleculeTable`
+const CLASS_ACTIONS = `${BASE_CLASS}-actions`
+const CLASS_MOBILE_MODE = `${BASE_CLASS}-mobile`
+const CLASS_MOBILE_MODE_ROW = `${CLASS_MOBILE_MODE}-row`
+const CLASS_MOBILE_MODE_CELL = `${CLASS_MOBILE_MODE_ROW}-cell`
+const CLASS_MOBILE_MODE_CELL_TITLE = `${CLASS_MOBILE_MODE_CELL}-title`
+const CLASS_MOBILE_MODE_CELL_CONTENT = `${CLASS_MOBILE_MODE_CELL}-content`
+
+const getActionsHeaderConfig = actions => ({
+  title: 'Actions',
+  key: 'actions',
+  align: 'left',
+  width: 150,
+  render: (text, record) => {
+    const extendedActions = React.cloneElement(actions, {
+      text,
+      record
+    })
+
+    return <span className={CLASS_ACTIONS}>{extendedActions}</span>
+  }
+})
+
+const renderMobileMode = ({title, dataSource, columns}) => { // eslint-disable-line
+  return (
+    <div className={CLASS_MOBILE_MODE}>
+      <h2>{title}</h2>
+      {dataSource.map((row, indexRow) => {
+        return (
+          <div key={indexRow} className={CLASS_MOBILE_MODE_ROW}>
+            {Object.keys(row).map((keyTitle, index) => {
+              const column = columns.filter(
+                column => column.key === keyTitle
+              )[0]
+
+              let dataCell = ''
+              if (typeof row[keyTitle] === 'object' && column.render) {
```

Comment:
Extract to a function:
```js
getDataCell({ column, rowContent }) {
  return typeof rowContent === 'object' && column.render
    ? column.render(rowContent)
    : rowContent 
}

// ...
getDataCell({ column, rowContent: row[keyTitle]})
```

Reply:
Done! Love your feedback. Suggestion and code ready to use 😁🔝